internal/pkg/source/source.go

```diff
@@ -213,3 +213,20 @@ func sourcesFromBlocks(fn *ssa.Function, conf classifier) []*Source {
 	}
 	return sources
 }
+
+func isProducedBySanitizer(v *ssa.Alloc, conf classifier) bool {
+	for _, instr := range *v.Referrers() {
+		store, ok := instr.(*ssa.Store)
```

Comment:
Would doing a `switch` here and acting only when the inst is a `*ssa.Call` read better?

Reply:
I think we need to cast into a `*ssa.Store` first, which if successful means the Call leads to a Store that refers to the Alloc, otherwise if the Alloc is referred to by a sanitizing Call at any point it will get identified as not a source (a few tests broke when I tried it, which is good). I think the code makes a decent job of describing that, and I don't see a way to make it more to the point. Do you think a comment would help?

Reply:
My bad - did not read this carefully enough.